installer/bundle/bundle_skel.sh

```diff
@@ -625,6 +625,7 @@ cd $EXTRACT_DIR
 # Do we need to remove the package?
 set +e
 if [ "$installMode" = "R" -o "$installMode" = "P" ]; then
+    rm -f /etc/cron.d/OMSConsistencyInvoker
```

Comment:
This line only deletes the cron file. It doesn't fix the verbose messages you mentioned. Since this is the first step in --purge and --remove and we still get the  messages, this cron file did not contribute to those messages in the first place.